infrastructure/application/Pulumi.staging.yaml

```diff
@@ -43,7 +43,8 @@ config:
   application:hasura-planx-api-key:
     secure: AAABANHLs3ItPxkteh0chwMP2bKuHO3ovuRLi4FsIrCqerzXVIaTLFDqNR+4KBTeMPz4cnF5tCTwsrJv9GruZdXU+lg=
   application:hasura-proxy-cpu: "512"
-  application:hasura-proxy-memory: "2048"
+  application:hasura-proxy-memory: "1024"
+  application:hasura-scaling-maximum: "2"
```

Comment:
Am I right in assuming these values need to be strings? So we should parse this as an int at read time, as below?

Reply:
This looks right to me! If it's not, we should get an obvious error on the staging deploy in the `pulumi up` step :ok_hand: